uv.lock

```diff
@@ -997,7 +998,6 @@ sdist = { url = "https://files.pythonhosted.org/packages/07/e9/ae44ea7d7605df9e5
 
 [[package]]
 name = "pymongo"
-version = "4.12.0.dev0"
```

Comment:
was this an intentional deletion? *curious*

Reply:
It was not, but is a welcome change because imo the local package version shouldn't be part of the lock file.

Reply:
Ah i see, in one of my previous tickets it was auto-deleted and i kept adding it back because I thought we needed it there HAHA

Reply:
Ah i see, in one of my previous tickets it was auto-deleted and i kept adding it back because I thought we needed it there HAHA